packages/cli/src/commands/dev.js

```diff
@@ -99,7 +108,7 @@ export const handler = async ({
     },
     web: {
       name: 'web',
-      command: `cd "${rwjsPaths.web.base}" && yarn cross-env NODE_ENV=development webpack serve --config "${webpackDevConfig}" ${forward}`,
+      command: `cd "${rwjsPaths.web.base}" && yarn cross-env NODE_ENV=development WATCH_NODE_MODULES=${watchNodeModules} webpack serve --config "${webpackDevConfig}" ${forward}`,
```

Comment:
```suggestion
      command: `cd "${rwjsPaths.web.base}" && yarn cross-env NODE_ENV=development RWJS_WATCH_NODE_MODULES=${watchNodeModules ? '1' : ''} webpack serve --config "${webpackDevConfig}" ${forward}`,
```

I prefix it with `RWJS` just to be on the safe side, you never know what people may name env-vars on their local machines.